index.js

```diff
@@ -1711,7 +1773,7 @@
   //. > S.Nothing.isNothing
   //. true
   //.
-  //. > S.Just(42).isNothing
+  //. > (S.Just (42)).isNothing
```

Comment:
This is a situation in which unnecessary parens are justified: `isNothing` in `S.Just (42).isNothing` is tightly bound, visually, to `42`.

Fortunately, such property access is rare in code written with Sanctuary (one could use `S.isNothing` in this case).